Gems/AWSCore/Code/Tests/Editor/Attribution/AWSCoreAttributionManagerTest.cpp

```diff
@@ -451,7 +458,10 @@ namespace AWSAttributionUnitTest
         // THEN
         AZStd::string serializedMetricValue = metric.SerializeToJson();
         ASSERT_TRUE(serializedMetricValue.find("\"o3de_version\":\"1.0.0.0\"") != AZStd::string::npos);
-        ASSERT_TRUE(serializedMetricValue.find(AZ::GetPlatformName(AZ::g_currentPlatform)) != AZStd::string::npos);
+        const auto platformValue = serializedMetricValue.find(expectedPlatform);
+        ASSERT_TRUE(platformValue != AZStd::string::npos);
```

Comment:
Nit: Maybe we can use ASSERT_NE(platformValue, AZStd::string::npos) here instead.